packages/codegen-ui-react/lib/forms/form-renderer-helper/all-props.ts

```diff
@@ -92,7 +92,7 @@ export const addFormAttributes = (
     }
 
     // TODO: Allow for other relationship types once valueMappings available
```

Comment:
Would appreciate it if you can delete this comment

Reply:
Got it. will delete in rev